src/main/java/org/jitsi/videobridge/rest/HandlerImpl.java

```diff
@@ -667,16 +670,18 @@ else if (RESTUtil.isJSONContentType(request.getContentType()))
                         }
                     }
                 }
-                if (status != 0)
+                else {
                     response.setStatus(status);
             }
```

Comment:
mis-aligned brace here?  also the opening brace for the else should be on a newline